wherescape/connectors/hubspot/hubspot_wrapper.py

```diff
@@ -0,0 +1,98 @@
+import logging
+import hubspot
+
+from hubspot.crm.properties import ApiException
+from .send_data import (
+    send_company_object,
+    send_contact_object,
+    send_deal_object,
+)
+
+"""
+module that makes sure the data is prepared correctly depending on where and how it will be send.
+Also provides the properties depending on the object.
+"""
+
+
+class Hubspot:
+    def __init__(self, access_token):
+        self.access_token = access_token
+        self.client = hubspot.Client.create(access_token=access_token)
+
+    def get_company_properties(self):
```

Comment:
I think you can remove the three functions: get_company_properties, get_deal_properties, get_contact_properties.
And directly use: get_object_properties!